remake_depth.yml

```diff
@@ -40,19 +40,21 @@ targets:
     command: depth_link_mnlakelist("data/depth_data_raw/depth_mn_lakelist.tsv")
     depends:
       - data/NHD_shape_large/NHDWaterbody.dbf
+      #- data/NHD_state_crosswalk/nhd2dowlknum.RData
     
   data/depth_data_linked/depth_herb_jacobson.tsv:
     command: depth_link_jacobson("data/depth_data_raw/depth_Herb_Jacobson.csv")
     depends:
       - data/NHD_shape_large/NHDWaterbody.dbf
+      #- data/NHD_state_crosswalk/nhd2dowlknum.RData
     
   data/depth_data_linked/depth_lagos.tsv:
     command: depth_link_lagos(necsc_config)
     depends:
       - data/NHD_shape_large/NHDWaterbody.dbf
     
   data/depth_data_linked/depth_swimms_wisconsin.csv:
-    command: depth_link_swimms("data/depth_data_raw/wisconsin_swims_lakes_depth.tsv")
+    command: depth_link_swimms("data/depth_data_raw/wisconsin_swims_lakes_depth.tsv", "data/NHD_state_crosswalk/nhd2wbic.RData")
```

Comment:
oh yeah, definitely. 

With these remake files I didn't yet know how to link to other remake targets from other files, but that really simplifies this kind of pattern... (e.g., https://github.com/jread-usgs/stream_metab_usa/blob/ec73e4578070b80405ff00fa1230f2f9898b1849/remake/4_data_release.yml#L4) 

Just a bit of work to clean these up and get them ready for prime time. Worth it?